onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -142,13 +142,23 @@ def compute_scale_zp(rmin, rmax, qmin, qmax, symmetric=False):
     rmin = min(rmin, 0)
     rmax = max(rmax, 0)
 
+    tiny_float = numpy.finfo(numpy.float32).tiny
+    if rmax < tiny_float:
+        rmax = 0
+    if rmin > -tiny_float:
+        rmin = 0
+
     if symmetric:
         absmax = max(abs(rmin), abs(rmax))
         rmin = -absmax
         rmax = +absmax
 
-    scale = (rmax - rmin) / float(qmax - qmin) if rmax != rmin else 1.0
-    zero_point = round(qmin - rmin / scale)
+    if rmax == rmin:
+        scale = 1.0
```

Comment:
is this meant to be if (rmax-rmin) <= tiny_float?

Reply:
Is it possible we have a corner case where rmax == rmin == 0.5 or some other normal fp32 value?

Reply:
No, there will be no this case because rmin <=0 and rmax >=0.